libbeat/publisher/pipeline/processor.go

```diff
@@ -53,7 +53,8 @@ func newProcessorPipeline(
 		localProcessors = makeClientProcessors(config)
 	)
 
-	needsCopy := global.alwaysCopy || localProcessors != nil || global.processors != nil
+	// needsCopy := global.alwaysCopy || localProcessors != nil || global.processors != nil
+	needsCopy := true
```

Comment:
This looks weird. Is the intention to always copy? Or can the `needsCopy` value change to false in other parts of the code?

Reply:
oh, some leftover from testing. :(